lib/carrierwave/uploader/cache.rb

```diff
@@ -178,7 +189,9 @@ def workfile_path(for_file=original_filename)
       alias_method :full_original_filename, :original_filename
 
       def cache_id=(cache_id)
-        raise CarrierWave::InvalidParameter, "invalid cache id" unless cache_id =~ /\A[\d]+\-[\d]+\-[\d]{4}\z/
+        # Earlier cache_id contained only 3 pieces, the "counter" part was introduced later.
```

Comment:
No need to talk about `later`. Change to "Earlier version used 3 part cache_id."